web-server-lib/web-server/http/request.rkt

```diff
@@ -34,6 +37,12 @@
    (values request? boolean?)))
 
 (provide/contract
+ [parse-bindings (-> bytes? (listof binding?))]
```

Comment:
It might be worth adding a comment here so that others don't make the same reasonable assumption you did in the future.

Reply:
Or maybe even document these bindings.